### PR TITLE
A Real(Native) Splash Screen

### PR DESCRIPTION
**Pull request by Fausto95**
splash-screen (1) (1)-min

- [ ] Already covered by tests
- [ ] Manual testing
- [ ] Unit testing

### CODE REVIEW

File: android/app/src/main/res/values/colors.xml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <color name="blue">#4481EB</color>
+    <color name="primary_dark">#4481EB</color>
```
Comment (VincentCATILLON) [inline android/app/src/main/res/values/colors.xml]: are you sure about these 2 values ? should not be `primary` and `primary_dark` ? and so avoid having 2 values ?

Comment (VincentCATILLON) [reply android/app/src/main/res/values/colors.xml]: this value is never used

File: src/components/carousel-card.js [hunk 2]
```diff
@@ -60,37 +59,28 @@ class CarouselCard extends React.Component<Props> {
...
+          </Html>
+        </View>
+      </View>
     );
   }
 }
 
 export {CarouselCard as Component};
-export default withLayout(CarouselCard);
+export default CarouselCard;
```
Comment (VincentCATILLON) [inline src/components/carousel-card.js]: These updates are not related to splashscreen.
Btw, 2 exports for the same thing

File: src/containers/carousel.js [hunk 3]
```diff
@@ -100,8 +108,8 @@ class Carousel extends React.PureComponent {
   extractKey = (item: Item) => item.iconName;
 
   render() {
-    const {state: {items}} = this;
-    const {width} = Dimensions.get('window');
+    const {state: {items}, props: {layout}} = this;
+    const width = layout && layout.width;
```
Comment (VincentCATILLON) [inline src/containers/carousel.js]: These updates are not related to splashscreen

File: android/app/src/main/res/values/colors.xml (hunk 1 above)
Comment (Fausto95) [reply android/app/src/main/res/values/colors.xml]: This is nor related to our code!
This is a react native work around!
